fvm/fvm_signature_test.go

```diff
@@ -721,7 +722,8 @@ func TestBLSMultiSignature(t *testing.T) {
 						)
 
 						err := vm.Run(ctx, script, view)
-						assert.Error(t, err)
+						assert.NoError(t, err)
+						assert.Error(t, script.Err)
```

Comment:
Due to the new `Unwrap` some errors are now user errors instead of implementation errors. We need to check if that is correct behavior in this case.

Reply:
As far as I understand the test case, which works with BLS signatures in a Cadence program, working with invalid BLS signatures should be a user error. Could you please add an assertion for the kind of error that is returned?

Reply:
I assumed that was the case, so I already did that.
```
assert.NoError(t, err) // < -- not implementation error
assert.Error(t, script.Err) // < -- but user error
```

It looks like this was previously reported incorrectly, and will now be ok.